queries/highlights.scm

```diff
@@ -1,132 +1,271 @@
-(comment) @comment
+; Includes
 
 [
-  "("
-  ")"
-  "{"
-  "}"
-  "["
-  "]"
-] @punctuation.bracket
+  "use"
+] @include
 
-[
-  ";"
-  "."
-  ","
-] @punctuation.delimiter
+; Keywords
 
 [
-  "if"
-  "ifdef"
-  "then"
-  "else"
-  "elseif"
-  "end"
-  "try"
-  "while"
-  "for"
-  "use"
-  "as"
-  "var"
-  "let"
-  "embed"
-  "fun"
-  "be"
-  "new"
+  "type"
   "actor"
   "class"
-  "struct"
   "primitive"
   "interface"
   "trait"
-  "type"
+  "struct"
+  "embed"
+  "let"
+  "var"
+  "be"
+  (compile_intrinsic)
+  "as"
+  "consume"
+  "object"
+  "where"
+] @keyword
+
+[
+  "fun"
+] @keyword.function
+
+[
+  "new"
+  "in"
```

Comment:
this is only used in `for` loops, maybe then it should also be part of `repeat`?

Reply:
no, it's typical to have such keywords like 'in' (similar to python's `for x in y`) as `@keyword.operator`

Reply:
If this is the case then lets go with this.